website_variants_extra/static/src/js/test_comments_qty.js

```diff
@@ -0,0 +1,43 @@
+(function(){
+    'use strict';
+    openerp.Tour.register({
+        id: 'shop_test_comment_qty',
+        name: 'Test Product Name with Variants',
+        path: '/shop/product/ipad-retina-display-5',
+        mode: 'test',
+        steps: [
+            {
+                title:     "Write first comment",
+                onload:    function(Tour){
+                               $('#comment textarea').text('1st Comment');
+                               $('#comment .fa-star-o:last').trigger('click');
+                            },
+                element:  'form a:contains(Post)',
+            },
+            {
+                title:     "Publish a comment",
+                onload:    function(Tour){
+                                $('#comments-list .btn-danger').first().trigger('click');
+                           },
+                waitFor:   '#comments-list:first:contains(1st)',
+                element:   '#comments-list',
+            },
+            {
+                title:     "Post second comment",
+                onload:    function(Tour){
+                               $('#comment textarea').text('2nd Comment');
+                               $('#comment .fa-star-o:last').trigger('click');
+                            },
+                waitFor:   '#comments-list:first:contains(1st)',
+                element:  'form a:contains(Post)',
+            },
+            {
+                title: 'check comment qty',
+                waitFor: '#no-reviews:contains(1)',
```

Comment:
@nhomar 
Esto debería probar la "cantidad" no solo que exista el elemento.:
it acctually tests "How many published comments are on the product"
`waitFor: '#no-reviews:contains(1)', ` <-- this code reffers to a label 'review {#no-reviews} comments' not if it exists, maybe you didn't quite understood the test, the only difference between your approach and what is done is the number, 1 or 6.